packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -194,6 +196,10 @@ export const CollectionScreenDetailsTile = ({
         dispatch(tracksActions.play())
         recordPlay(playingTrackId)
       } else if (trackCount > 0 && firstTrack) {
+        // TODO: This doesn't work.
+        // Need to clear the redux queue and also the react-native-track-player queue
+        dispatch(queueActions.clear({}))
+        TrackPlayer.reset()
```

Comment:
@dylanjeffers could use a little help here. This approach works for web, but I'm not sure how to appropriately clear the queue for native since we have both the redux queue and the react-native-track-player queue.

Is there a simple way to clear everything and re-enqueue from the lineup?